parlai/crowdsourcing/tasks/acute_eval/webapp/__init__.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
Nit: no need for init in webapp directory (especially as you have modified the test already).